service/src/main/java/aero/minova/cas/sql/SqlUtils.java

```diff
@@ -127,4 +134,106 @@ public static Value parseSqlParameter(CallableStatement statement, int index, Co
 		}
 	}
 
+	public static Table convertSqlResultToTable(Table inputTable, ResultSet sqlSet, Logger logger, Object conversionUser) {
+		try {
+			Table outputTable = new Table();
+			outputTable.setName(inputTable.getName());
+			outputTable.setColumns(//
+					inputTable.getColumns().stream()//
+							.filter(column -> !Objects.equals(column.getName(), Column.AND_FIELD_NAME))//
+							.collect(Collectors.toList()));
+			while (sqlSet.next()) {
+				outputTable.addRow(SqlUtils.convertSqlResultToRow(outputTable, sqlSet, logger, conversionUser));
+			}
+			return outputTable;
+		} catch (Throwable e) {
+			throw new RuntimeException(e);
+		}
+	}
+
+	/**
+	 * Das Prepared Statement wird mit den dafür vorgesehenen Parametern befüllt. Diese werden aus der übergebenen inputTable gezogen.
+	 *
+	 * @param inputTable
+	 *            die Table, welche vom getIndexView aufgerufen wurde
+	 * @param preparedStatement
+	 *            das Prepared Statement, welches nur noch befüllt werden muss
+	 * @param query
+	 *            Das bereits fertig aufgebaute Sql Statement, welches statt der Werte '?' enthält. Diese werden hier 'ersetzt'.
+	 * @param sb
+	 *            Ein StringBuilder zum Loggen der inputParameter.
+	 * @param Logger
+	 *            Ein Logger, welcher bei Fehlern die Exception loggen kann.
+	 * @return das befüllte, ausführbare Prepared Statement
+	 */
+	public static PreparedStatement fillPreparedViewString(Table inputTable, CallableStatement preparedStatement, String query, StringBuilder sb,
```

Comment:
Vielleicht eher fillpreparedViewMssqlString nennen oder in eine MssqlUtils-Klasse packen?

Reply:
> Vielleicht eher fillpreparedViewMssqlString nennen oder in eine MssqlUtils-Klasse packen?

Er wird aber auch im JOOQService verwendet. 😅